setup.py

```diff
@@ -62,6 +62,6 @@
     test_suite="tests",
     tests_require=test_requirements,
     url="https://github.com/VulcanClimateModeling/fv3config",
```

Comment:
Here too?

Reply:
although that will be changed by end of summer